pkg/metrics/collector.go

```diff
@@ -96,21 +102,254 @@ func (c *metricsCollector) publishVolumeStats(ctx context.Context, volume *types
 	)
 }
 
+type DriveStats struct {
+	ReadSectors  int64
+	ReadTicks    int64
+	WriteSectors int64
+	WriteTicks   int64
+	TimeInQueue  int64
+}
+
+func (c *metricsCollector) publishDriveStats(ctx context.Context, drive *types.Drive, ch chan<- prometheus.Metric) {
+	device, err := c.getDeviceByFSUUID(drive.Status.FSUUID)
+	if err != nil {
+		klog.ErrorS(
+			err,
+			"unable to find device by FSUUID; "+
+				"either device is removed or run command "+
+				"`sudo udevadm control --reload-rules && sudo udevadm trigger`"+
+				" on the host to reload",
+			"FSUUID", drive.Status.FSUUID)
+		client.Eventf(
+			drive, client.EventTypeWarning, client.EventReasonMetrics,
+			"unable to find device by FSUUID %v; "+
+				"either device is removed or run command "+
+				"`sudo udevadm control --reload-rules && sudo udevadm trigger`"+
+				" on the host to reload", drive.Status.FSUUID)
+
+		return
+	}
+
+	// Online/Offline Status
+	if _, err := os.Stat("/sys/block/" + device); os.IsNotExist(err) {
+		// Drive is offline
+		ch <- prometheus.MustNewConstMetric(
+			prometheus.NewDesc(
+				prometheus.BuildFQName(consts.AppName, "stats", "drive_status"),
```

Comment:
can we have `_drive_ready` instead of `_drive_status`? 

if these directories are not present, then it means the drive is "not ok" (or) "not ready". we can have a label name accordingly.